flexget/plugins/modify/manipulate.py

```diff
@@ -120,27 +122,31 @@ def process(self, entry, jobs):
                     from_field,
                     field_value,
                 )
-
                 if config.get('remove'):
                     if field in entry:
                         del entry[field]
                         modified = True
                     continue
-
                 if 'extract' in config:
                     if not field_value:
-                        logger.warning('Cannot extract, field `{}` is not present', from_field)
+                        logger.warning('Cannot extract, field `%s` is not present', from_field)
```

Comment:
we use [loguru](https://github.com/Delgan/loguru) as our logger, so please retain the usage of `{}` for logging

Reply:
Apologies I didn't notice! They should all be in the loguru style now.